tests/testthat/helper-ckanr.R

```diff
@@ -153,9 +153,7 @@ check_organization <- function(url, x){
   }
 }
 
-u <- get_test_url()
```

Comment:
If we were to let get_test_url() have a default value that works here (i.e. "http://localhost:5000"), we could remove the environment secret.